Added scientific calculator

Neelima-git opened the pull request:
<!-- Remove the sections which are not applicable -->

### 🛠️ Fixes Issue (Number)

Closes #222 

### 👨‍💻 Changes proposed

Added scientific calculator

### ✅ Check List (Check all the applicable boxes) <!-- Follow the below conventions to check the box -->

- [x] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [x] This PR does not contain plagiarized content.
- [x] My Addition/Changes works properly and matches the overall repo pattern.
- [x] The title of my pull request is a short description of the requested changes.

### 📄 Note to reviewers

Please let me know if changes are required. 

### 📷 Screenshots

![scientific_calculator](https://user-images.githubusercontent.com/67384625/156929644-6a92bb49-8ac1-46d8-bcb6-86fc6f408472.png)

Astrodevil commented:
Your commits on index.html creates error in last 3 listed projects of landing page. Please check it once properly. Also add your project in index.html

Neelima-git commented:
> Your commits on index.html creates error in last 3 listed projects of landing page. Please check it once properly. Also add your project in index.html

I looked into the issue. The images of the last three projects were missing in my version. I added them and fixed it! I also updated my project in index.html. Hope everything is correct now. If any other modifications are required, please do let me know!
